Frontend/src/components/Research.jsx

```
@@ -0,0 +1,330 @@
+  
+        alert("Paper uploaded successfully!");
+        setTitle("");
+        setAuthor("");
+        setResearchField("");
+        setFile(null);
+      } catch (error) {
+        if (error.response) {
+          console.error("Upload Error:", error.response.data);
+          alert("Upload failed: " + JSON.stringify(error.response.data));
+        } else {
+          alert("An error occurred while uploading.");
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)